### ISSUE TITLE
ansible-lint 4.3.0 fails with Homebrew Python 3.8 on macOS

### ISSUE THREAD

**Issue by sfuerte** [ansible/ansible-lint#974]
##### Summary

Started to get the traceback after upgrading to the latest version:
```
+ ansible-lint --nocolor -p --parseable-severity --exclude <home>/.ansible/roles/
Traceback (most recent call last):
  File "/usr/local/bin/ansible-lint", line 8, in <module>
    sys.exit(main())
  File "/usr/local/Cellar/ansible-lint/4.3.0/libexec/lib/python3.8/site-packages/ansiblelint/__main__.py", line 121, in main
    runner = Runner(rules, playbook, options.tags,
  File "/usr/local/Cellar/ansible-lint/4.3.0/libexec/lib/python3.8/site-packages/ansiblelint/runner.py", line 43, in __init__
    self._update_exclude_paths(exclude_paths)
  File "/usr/local/Cellar/ansible-lint/4.3.0/libexec/lib/python3.8/site-packages/ansiblelint/runner.py", line 52, in _update_exclude_paths
    paths = ansiblelint.utils.expand_paths_vars(exclude_paths)
  File "/usr/local/Cellar/ansible-lint/4.3.0/libexec/lib/python3.8/site-packages/ansiblelint/utils.py", line 783, in expand_paths_vars
    paths = [expand_path_vars(p) for p in paths]
  File "/usr/local/Cellar/ansible-lint/4.3.0/libexec/lib/python3.8/site-packages/ansiblelint/utils.py", line 783, in <listcomp>
    paths = [expand_path_vars(p) for p in paths]
  File "/usr/local/Cellar/ansible-lint/4.3.0/libexec/lib/python3.8/site-packages/ansiblelint/utils.py", line 775, in expand_path_vars
    path = path.strip()
AttributeError: 'PosixPath' object has no attribute 'strip'
```

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details
```console (paste below)
> ansible --version
ansible 2.9.11
  config file = <home>/projects/<project_path>/ansible.cfg
  configured module search path = ['<home>/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/2.9.11/libexec/lib/python3.8/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.5 (default, Jul 21 2020, 10:48:26) [Clang 11.0.3 (clang-1103.0.32.62)]

> ansible-lint --version
ansible-lint 4.3.0
```

- ansible installation method: Homebrew on macOS
- ansible-lint installation method: Homebrew on macOS

```
> brew list | grep ansible
ansible
ansible-lint
```

##### OS / ENVIRONMENT
```
> system_profiler SPSoftwareDataType

    System Software Overview:

      System Version: macOS 10.15.6 (19G2021)
      Kernel Version: Darwin 19.6.0
```

##### STEPS TO REPRODUCE
upgraded to the latest version

**Comment by mazerty** [comment]
i'm also having the same error running molecule 2.22 with a brand new virtualenv
```
--> Executing Ansible Lint on /home/indus/workspace/RECETTE/molecule/molecule/default/playbook.yml...
Traceback (most recent call last):
  File "/home/indus/workspace/RECETTE/molecule/.venv/bin/ansible-lint", line 11, in <module>
    sys.exit(main())
  File "/home/indus/workspace/RECETTE/molecule/.venv/lib64/python3.6/site-packages/ansiblelint/__main__.py", line 123, in main
    options.verbosity, checked_files)
  File "/home/indus/workspace/RECETTE/molecule/.venv/lib64/python3.6/site-packages/ansiblelint/runner.py", line 43, in __init__
    self._update_exclude_paths(exclude_paths)
  File "/home/indus/workspace/RECETTE/molecule/.venv/lib64/python3.6/site-packages/ansiblelint/runner.py", line 52, in _update_exclude_paths
    paths = ansiblelint.utils.expand_paths_vars(exclude_paths)
  File "/home/indus/workspace/RECETTE/molecule/.venv/lib64/python3.6/site-packages/ansiblelint/utils.py", line 783, in expand_paths_vars
    paths = [expand_path_vars(p) for p in paths]
  File "/home/indus/workspace/RECETTE/molecule/.venv/lib64/python3.6/site-packages/ansiblelint/utils.py", line 783, in <listcomp>
    paths = [expand_path_vars(p) for p in paths]
  File "/home/indus/workspace/RECETTE/molecule/.venv/lib64/python3.6/site-packages/ansiblelint/utils.py", line 775, in expand_path_vars
    path = path.strip()
AttributeError: 'PosixPath' object has no attribute 'strip'
```
ansible-lint is a soft dependency of molecule, and by forcing the version to 4.2.0 it's working again

**Comment by sfuerte** [comment]
No even `virtualenv` in my case: 

```
> /usr/local/bin/pip3.8 list
Package          Version
---------------- ---------
awscli           1.18.122
awscli-login     0.1.0a6
beautifulsoup4   4.9.1
boto             2.49.0
boto3            1.14.45
botocore         1.17.45
bs4              0.0.1
certifi          2020.6.20
cffi             1.14.2
chardet          3.0.4
colorama         0.4.3
crypto           1.4.1
cryptography     3.0
daemoniker       0.2.3
docker           4.3.0
docutils         0.16
idna             2.10
jmespath         0.10.0
keyring          21.3.0
lxml             4.5.2
Naked            0.1.31
ntlm-auth        1.5.0
pip              20.2.2
psutil           5.7.2
pyasn1           0.4.8
pycparser        2.20
pycryptodomex    3.9.8
python-dateutil  2.8.1
PyYAML           5.3.1
requests         2.24.0
requests-ntlm    1.1.0
rsa              4.6
s3transfer       0.3.3
scapy            2.4.3
setuptools       49.6.0
shellescape      3.8.1
six              1.15.0
soupsieve        2.0.1
urllib3          1.25.10
websocket-client 0.57.0
wheel            0.35.1
```

and packages inside `ansible-lint` Cellar:
```
> ls -l /usr/local/Cellar/ansible-lint/4.3.0/libexec/lib/python3.8/site-packages/
total 1520
drwxr-xr-x   9 user  staff     288 Aug 18 04:58 Jinja2-2.11.2-py3.8.egg-info
drwxr-xr-x   7 user  staff     224 Aug 18 04:58 MarkupSafe-1.1.1-py3.8.egg-info
drwxr-xr-x   7 user  staff     224 Aug 18 04:58 PyYAML-5.3.1-py3.8.egg-info
drwxr-xr-x   3 user  staff      96 Aug 19 11:22 __pycache__
-rwxr-xr-x   1 user  staff  152196 Aug 18 04:58 _cffi_backend.cpython-38-darwin.so
drwxr-xr-x   4 user  staff     128 Aug 18 04:58 _distutils_hack
-rwxr-xr-x   1 user  staff  326380 Aug 18 04:58 _ruamel_yaml.cpython-38-darwin.so
-rwxr-xr-x   1 user  staff  242972 Aug 18 09:07 _yaml.cpython-38-darwin.so
drwxr-xr-x  22 user  staff     704 Aug 19 11:22 ansible
drwxr-xr-x   9 user  staff     288 Aug 18 04:58 ansible-2.9.12-py3.8.egg-info
drwxr-xr-x  11 user  staff     352 Aug 18 04:58 ansible_lint-4.3.0.dist-info
drwxr-xr-x   6 user  staff     192 Aug 18 04:58 ansible_test
drwxr-xr-x  17 user  staff     544 Aug 19 11:22 ansiblelint
drwxr-xr-x  22 user  staff     704 Aug 18 04:58 cffi
drwxr-xr-x  10 user  staff     320 Aug 18 04:58 cffi-1.14.2-py3.8.egg-info
drwxr-xr-x  10 user  staff     320 Aug 19 11:22 cryptography
drwxr-xr-x  14 user  staff     448 Aug 18 04:58 cryptography-3.0.dist-info
-rw-r--r--   1 user  staff     131 Aug 18 04:58 distutils-precedence.pth
-rw-r--r--   1 user  staff     126 Aug 18 04:58 easy_install.py
drwxr-xr-x  30 user  staff     960 Aug 19 11:22 jinja2
drwxr-xr-x   9 user  staff     288 Aug 19 11:22 markupsafe
drwxr-xr-x   6 user  staff     192 Aug 18 04:58 pip
drwxr-xr-x   9 user  staff     288 Aug 18 04:58 pip-20.2.2.dist-info
drwxr-xr-x   6 user  staff     192 Aug 19 11:22 pkg_resources
drwxr-xr-x  15 user  staff     480 Aug 18 04:58 pycparser
drwxr-xr-x   7 user  staff     224 Aug 18 04:58 pycparser-2.20-py3.8.egg-info
drwxr-xr-x   3 user  staff      96 Aug 18 04:58 ruamel
-rw-r--r--   1 user  staff     539 Aug 18 04:58 ruamel.yaml-0.16.10-py3.8-nspkg.pth
drwxr-xr-x   9 user  staff     288 Aug 18 04:58 ruamel.yaml-0.16.10-py3.8.egg-info
drwxr-xr-x   8 user  staff     256 Aug 18 04:58 ruamel.yaml.clib-0.2.0-py3.8.egg-info
drwxr-xr-x  43 user  staff    1376 Aug 18 09:07 setuptools
drwxr-xr-x  11 user  staff     352 Aug 18 04:58 setuptools-49.6.0.dist-info
drwxr-xr-x   7 user  staff     224 Aug 18 04:58 six-1.15.0-py3.8.egg-info
-rw-r--r--   1 user  staff   34159 Aug 18 04:58 six.py
drwxr-xr-x  12 user  staff     384 Aug 18 04:58 wheel
drwxr-xr-x   9 user  staff     288 Aug 18 04:58 wheel-0.35.1.dist-info
drwxr-xr-x  20 user  staff     640 Aug 19 11:22 yaml
```

**Comment by ssbarnea** [comment]
A fix for this was included in 4.3.1
